.travis.yml

```diff
@@ -1,72 +1,72 @@
-version: ~> 1.0
-dist: focal
-# this arch is required as is for Partner Queue Solution - DO NOT MODIFY
-arch: ppc64le
-language: java
-
-cache:
-  directories:
-    - ~/.m2
-
-before_install:
-  - sudo apt update
-
-branches:
-  only:
-    - master
-
-install:
-  - sudo apt install -y groovy xmlstarlet ant-optional ruby
-  - gem install mdl
-
-
-jobs:
-  fast_finish: true
-  include:
-    - jdk: openjdk11
-      env:
-        - DESC="markdown lint"
-        - CMD="./.ci/travis.sh markdownlint"
-
-    - jdk: openjdk11
-      env:
-        - DESC="releasenotes-builder"
-        - CMD="./.ci/travis.sh releasenotes-builder"
-
-    - jdk: openjdk11
-      env:
-        - DESC="patch-diff-report-tool"
-        - CMD="./.ci/travis.sh patch-diff-report-tool"
-
-    - jdk: openjdk11
-      env:
-        - DESC="checkstyle-tester (diff.groovy) on linux"
-        - CMD="./.ci/travis.sh checkstyle-tester-diff-groovy-patch"
-
-    - jdk: openjdk11
-      env:
-        - DESC="checkstyle-tester (diff.groovy) on linux with base and patch configs"
-        - CMD="./.ci/travis.sh checkstyle-tester-diff-groovy-base-patch"
-
-    - jdk: openjdk11
-      env:
-        - DESC="checkstyle-tester (diff.groovy) on linux with enabled patchOnly"
-        - CMD="./.ci/travis.sh checkstyle-tester-diff-groovy-patch-only"
-
-    - jdk: openjdk11
-      env:
-        - DESC="codenarc validation for groovy files"
-        - CMD="./.ci/travis.sh codenarc"
-
-script:
-  # required to run javadoc in travis
-  # this must be in yml to pass to other scripts
-  - export JAVA_HOME="/usr/lib/jvm/"$(ls /usr/lib/jvm/ | head -n 1)"/"
-  - echo "JAVA_HOME="$JAVA_HOME
-  - export PATH=$JAVA_HOME:$PATH
-  - echo "PATH="$PATH
-  - ./.ci/travis.sh init-m2-repo
-  - ./.ci/travis.sh run-command "$CMD"
-  - sleep 5s
-
-after_success:
+#version: ~> 1.0
```

Comment:
Not sure what to do with this file, we can just delete it or I can rename it.

Reply:
I assume we are talking about `.travis.yml`. If we are disabling travis completely, we should just delete it. If we need to restore it, we can go back in the git history.

Reply:
deleted